docs/use_cases/recommender_systems.md

```diff
@@ -1,32 +1,31 @@
 
-(We deliberately took a detour and observed the problem via graphs. Note that this approach is known as ‘joint matrix factorization’ in the literature).
-
-Once the problem is understood as inherently a graph problem, it is easy to see how additional information can be added. What we want to achieve is to somehow let the algorithm know about the new links, which would help group similar items or users together. In other words, we want to ‘inject’ new nodes, which would link the nodes belonging to the same group. How could this be achieved? Have a look at illustration below:
+Once the problem is understood as inherently a graph problem, it is easy to see how the model can be extended to include additional metadata. What we want to achieve is to somehow let the algorithm know about the new links, which would help group similar items or users together. In other words, we want to ‘inject’ new nodes, which would link the nodes belonging to the same group. How could this be achieved? Have a look at illustration below:
 
 ![](assets/use_cases/recommender_systems/dummy_nodes.jpg)
 
-There are three users, U1, U2 and U3, and four items, I1, I2, I3, I4. The user U1 has interacted with items I1 and I3. Now, there is a Dummy User, who links the items that have the same color (I1, I3, I4). 
-The dummy user couples similar items, helping the model identify related content. This increases the chances of item I4 to be recommended to user U1.
+There are three users, u1, u2 and u3, and four items, i1, i2, i3, i4. The user U1 has interacted with items i1 and i3. Now, there is a Dummy User, who links the items that have the same color (i1, i3, i4). 
+The dummy user couples similar items, helping the model identify related content. This increases the chances of item i4 to be recommended to user u1.
 
 **Adaptation**
 
 As discussed above, we only need to add dummy data: in case of only item side information available, add dummy users; in case of only user side information available, add dummy items; in case both the user and the item information is available, add both the dummy users and the dummy items. Obviously, the dummy nodes should not be part of recommendations; these nodes are only there to help ‘inject’ some ‘commonality’ of the nodes belonging to a certain group.
```

Comment:
previous comments - not yet addressed:

"dummy data" is not discussed above; only a dummy user.

"As discussed above" means you've already discussed it, which you haven't. Instead of using this phrase, say exactly what you mean. It may take more effort, but it will help clarify your thinking.

instead of "in case of only item side information availabile..."
say "when only item side information is available..."